docs/usage/api-reference.md

```diff
@@ -6,6 +6,6 @@
 
    api/js-api.md
    api/python-api.md
-   api/micropip-api.md
+   Micropip API <https://micropip.pyodide.org/en/stable/project/api.html>
```

Comment:
Or we just remove it, not sure

Reply:
Maybe we should also link to specific version of micropip here? If not, when we release a new micropip version, old Pyodide versions will have old micropip but will be linked to latest micropip docs.

Reply:
Perhaps we can parse micropip version directly from meta.yaml file. It is quite hacky, but we are already doing something similar [here](https://github.com/pyodide/pyodide/blob/62dcf9119f406fe551390b899f0a0f0a5b3529fc/docs/sphinx_pyodide/sphinx_pyodide/packages.py#L25-L26)

Reply:
Thanks for the pointer. That's a possibility but I don't really want to create a sphinx plugin just for this. Ideally, users would never see this page once we figure out redirects. All the other links should redirect to the matching micropip version. Overall older versions of Pyodide should work with newer versions of micropip, so it's probably not that critical.

Reply:
But if people never actually see this page for some reason then I agree it's not a big deal...

Reply:
Actually, just sphinx ref to `micropip` should link to the correct version, without any hacks, as we are doing in https://pyodide.org/en/latest/usage/loading-packages.html#installing-packages 

Will fix in another PR